Viper reference removed

Shuffla opened the pull request.

Shuffla committed:
Polish of files, inital work on Cormorant story

Shuffla committed:
Small polish pass and an update

Polished Pheasant, and updated Cormorant.

Shuffla committed:
Merge branch 'main' into chris

Shuffla committed:
Change Cormorant to motor convoy instead of train :: Chris

Shuffla committed:
Cleanup of Viper reference

Commit:
Merge branch 'main' into jonas

Commit:
Swallow & Pheasant improvements

- Swallow :: no longer requires close escort
- Swallow :: can now be ordered to hold
- Swallow :: changes to make it more generic, and stand-alone from OVS
- Pheasant :: convoy now has #DCAF.Mobiledefense behaviour

Commit:
Minor improvements to Pheasant story

- Included mobile defense MANPADS as units in Pheasant.miz (to carry over in STM)
- Minor bug fixes to story

jonasrembratt reviewed (commented):
Please remove Swallow and Pheasant stories from PR (changes was already implemented in other branches)

Shuffla committed:
Update to PR

Removed local versions of Swallow/Pheasant, edited README, added Albatroz and Cormorant

Shuffla committed:
Revert "Cleanup of Viper reference"

This reverts commit 03e3773bf92d401f27ec0c2befc9465801529f80.

Shuffla committed:
Revert "Small polish pass and an update"

This reverts commit 6ddf1d70d83d60fe39cbf4c18ddf80f28f1ce308.

Shuffla committed:
Revert "Polish of files, inital work on Cormorant story"

This reverts commit 3b05c0bf3a6bc317d0f15495f690e9646f00dc91.

Shuffla committed:
Merge remote-tracking branch 'origin/jonas' into chris

Shuffla commented:
I merged your changes into my PR, and resubmitted. @jonasrembratt
